src/components/Cards/VotingTableCard/VotingTableCard.stories.mdx

```diff
@@ -0,0 +1,134 @@
+import { action } from "@storybook/addon-actions";
+import { text, array, boolean } from "@storybook/addon-knobs";
+import { Meta, Props, Story } from "@storybook/addon-docs/blocks";
+import VotingTableCard from "./VotingTableCard";
+
+<Meta title="Library/Tables/VotingRecordTable" component={VotingTableCard} />
+
+### An Empty Table
+
+<Props of={VotingTableCard} />
+
+<Story name="empty table">
```

Comment:
This is a minor nitpick but could we capitalize all the story names just to be consistent with the rest of the component library/stories?

Reply:
Yep, done.